xml/System.Windows.Forms/CloseReason.xml

```diff
@@ -149,7 +149,8 @@
       </ReturnValue>
       <MemberValue>4</MemberValue>
       <Docs>
-        <summary>The Microsoft Windows Task Manager is closing the application.</summary>
+        <summary><para>The Microsoft Windows Task Manager is closing the application.</para>
+        <para>In Windows 8.1 and later versions, this member is not used, because Task Manager issues the `WM_SYSCOMMAND` message with `SC_CLOSE`. In Windows 7 and earlier versions, the `WM_CLOSE` message was issued. This action is now indistinguishable from `CloseReason.UserClose`.</para></summary>
```

Comment:
```suggestion
        <para>In Windows 8.1 and later versions, this member is not used, because Task Manager issues the `WM_SYSCOMMAND` message with `SC_CLOSE`. In Windows 7 and earlier versions, the `WM_CLOSE` message was issued. This action is now indistinguishable from and misclassified as  `CloseReason.UserClosing`.</para></summary>
```

Reply:
@gewarren, it looks like you made another change, but I cannot commit if for some reason.
I did notice that for some reason you have 2 spaces after "and misclassified as".

Reply:
Thanks - I went ahead and committed this one myself to save a step. The double space is okay (probably not worth doing another build to fix it).